docs/manage_infrastructure.md

```diff
@@ -425,6 +425,32 @@ It works even if the VM is running, because we'll automatically export a snapsho
 
 In the VM "Snapshots" tab, you can also export a snapshot like you export a VM.
 
+## VM migration
+
+### Simple VM Migration (VM.pool_migrate)
+
+Simple migration involves moving the VM without moving its disks. This feature is only possible when the VM's disks are on a shared SR by both hosts.
+
+#### Use Case
+
+- Move a VM within the same pool from host A to host B without moving the VM's VDIs.
+
+### VM Migration with Storage Motion (VM.migrate_send)
+
+VM migration with storage motion allows you to move a VM from one host to another when the VM's disks are not on a shared SR between the two hosts or if a specific network is chosen for the migration.
+
+#### Use Cases
+
+- Migrate a VM to another pool.
+- Migrate a VM within the same pool from host A to host B by selecting a network for the migration.
+- Migrate a VM within the same pool from host A to host B by moving the VM's VDIs to another storage.
+
+### Expected Behavior
+
+- Migrating a VM that has VDIs on a shared SR from host A to host B must trigger a "Simple VM Migration".
+- Migrating a VM that has VDIs on a shared SR from host A to host B using a particular network must trigger a "VM Migration with Storage Motion" without moving its VDIs.
```

Comment:
So the network will be used to transfer the RAM content, right?

Reply:
Yes.